tests/test.py

```diff
@@ -154,6 +158,11 @@ def _run_tests(app_url, app_args, expected_output, args={}, config={}):
 
 
 def _submit_job(app_url, app_args, args={}, config={}):
+    if os.environ.get('SECURITY') == 'strict':
```

Comment:
let's factor this out into an `_is_strict()` function, and replace all 3 occurrences